README.md

```diff
@@ -39,42 +55,111 @@ Install `dlv` via `go get -u github.com/go-delve/delve/cmd/dlv`
 2. Install CRDs via `make install`
 3. Start VSCode debugging session by selecting `Meteor Operator` profile
 
-### Run from local machine without debugging
-
-1. Log in to your cluster via `oc login`
-2. Run `make install run`
+## Quick build and deploy to a cluster
 
-## Quick deploy via operator-sdk
+The following commands will build the image, push it to the container registry, and deploy the operator to the `meteor-system` namespace of your currently configured cluster (as per your `$KUBECONFIG`):
```

Comment:
Another reference to a namespace change that reverted, so this also needs to revert:

```suggestion
The following commands will build the image, push it to the container registry, and deploy the operator to the `aicoe-meteor` namespace of your currently configured cluster (as per your `$KUBECONFIG`):
```